gcp-cloud-run/build.gradle

```diff
@@ -0,0 +1,56 @@
+plugins {
+    id 'com.google.cloud.tools.jib'
+    id 'org.springframework.boot'
+}
+apply from: project(":").file('gradle/java.gradle')
+
+dependencies {
+    implementation project(':local-rest-scorer')
+    implementation group: 'org.slf4j', name: 'slf4j-api'
+    implementation group: 'com.google.cloud', name: 'google-cloud-storage'
+}
+
+test {
+    useJUnitPlatform()
+}
+
+bootRun {
+    systemProperties System.properties
+}
+
+bootJar {
+    mainClassName = 'ai.h2o.mojos.deploy.gcp.cloud.run.GcpScorerApplication'
+}
+
+// Include the local rest scorer executable jar in the root package distribution zip file.
+rootProject.distributionZip {
+    dependsOn bootJar
+    into(this.project.path.replace(":", "/")) {
+        from bootJar.archivePath
+    }
+}
+
+// Docker image configuration.
+jib {
+    extraDirectories {
```

Comment:
yes it technically is. Some weird issue with `jib` from what I could find... regarding this and the below comment. One of the jib issues I found while googling up on this: https://github.com/GoogleContainerTools/jib/issues/954. But long story short is that jib doesn't use the springboot bootjar as one would expect, but takes the `.class` files created inside the `build` dir of the project. So the required `local-rest-scorer` `.class` files don't get added to the classpath and things don't work. 

Side Note: we could use arg `containerizingMode = 'packaged'` which essentially places the actual boot jar into the classpath and we can then override the entrypoint, but it increases the size of the resultant docker image by nearly 30% so I figured this to be the less impactful method.

Reply:
I think we need to mark the `bootJar` in `local-rest-scorer` as `archiveClassifier = "boot"` so that the dependency in the gcp one actually brings in the plain jar as opposed to the bootJar (which has a special form and the `ScorerApplication` class is well hidden deep inside it).

I tried it removing the `extraDirectories` and `extraClasspath` and it seems to work.

I would personally rather use that than the `packaged` mode. Not pushing though.

More context: https://docs.spring.io/spring-boot/docs/current/gradle-plugin/reference/html/#packaging-executable-and-normal

Reply:
ahh this looks interesting. Let me try quickly. If it works I will push a change before the merge.

Reply:
@osery this worked great! can't believe I didn't notice this. Thanks! Made change to remove `extraDirectories` and `extraClasspath` values and added `archiveClassifier = 'boot'` to local-rest-scorer `build.gradle`

ran the docker image for local rest scorer and updated readme with correct jar for local run.